docs/concepts/overview/working-with-objects/common-labels-annotations.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Common Label and Annotation Recommendations
+---
+
+{% capture overview %}
+Many tools can be used to visualize and manage Kubernetes objects. This goes beyond
+tools such as dashboard or kubectl and includes many tools in the Kubernetes ecosystem.
+To enable interoperability between tools, a common set of labels and annotation
+will allow objects to be described in a common manner that all tools can understand.
```

Comment:
Minor wording change: recommend removing “will” - make the sentence direct and avoids future-tensing. 

recommened change:
> “will allow” to “allows”